app/views/mass_rollouts/review_deploy.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= page_title "Mass Rollout for #{@deploy_group.name} in #{@deploy_group.environment.name}" %>
+
+<% if @stages.empty? %>
+  <p>There are no stages to deploy for this Deploy Group.</p>
+<% else %>
+  <%= form_tag deploy_deploy_group_mass_rollouts_path(@deploy_group) do %>
+    <small>Selected stages will be deployed.</small>
+    <section class="clearfix">
+      <div class="table table-hover table-condensed">
+        <table class="table">
+          <thead>
+          <tr>
+            <th></th>
+            <th>Project</th>
+            <th>Stage</th>
+          </tr>
```

Comment:
Ideally would be good to show the reference that would be deployed here. (maybe another TODO)

Reply:
yeah, todo